process_report/processors/prepayment_processor.py

```diff
@@ -201,12 +220,39 @@ def _apply_prepayments(self):
                         debit_entry_mask, invoice.PREPAY_DEBIT_FIELD
                     ] = prepay_amount_used
 
+    def _get_prepay_credits_snapshot(self):
+        managed_groups_list = list()
+        for group_name, group_dict in self.group_info_dict.items():
+            if group_dict[invoice.PREPAY_MANAGED_FIELD]:
+                managed_groups_list.append(group_name)
+
+        credits_mask = (
+            self.prepay_credits[invoice.PREPAY_MONTH_FIELD] == self.invoice_month
+        ) & (
+            self.prepay_credits[invoice.PREPAY_GROUP_NAME_FIELD].isin(
+                managed_groups_list
+            )
+        )
+        return self.prepay_credits[credits_mask]
+
     def _backup_s3_prepay_debits(self):
         invoice_bucket = util.get_invoice_bucket()
         invoice_bucket.upload_file(
             self.prepay_debits_filepath, self.PREPAY_DEBITS_S3_BACKUP_FILEPATH
         )
 
+    def _export_prepay_credits_snapshot(self, credits_snapshot):
+        credits_snapshot.to_csv(self.CREDITS_SNAPSHOT_FILEPATH, index=False)
+
+    def _export_s3_prepay_credits_snapshot(self, credits_snapshot):
+        invoice_bucket = util.get_invoice_bucket()
+        invoice_bucket.upload_file(
+            self.CREDITS_SNAPSHOT_FILEPATH, self.CREDITS_SNAPSHOT_S3_FILEPATH
```

Comment:
you changed the capitalization, but didn't change the capitalization of where they are used.